library/src/main/java/com/novoda/downloadmanager/lib/FileDownloadInfo.java

```diff
@@ -356,6 +356,10 @@ private void clearHeaders() {
         requestHeaders.clear();
     }
 
+    public boolean isResumable() {
+        return alwaysResume || (eTag != null && isNoIntegrity());
```

Comment:
https://github.com/novoda/download-manager/blob/master/library/src/main/java/com/novoda/downloadmanager/lib/DownloadThread.java#L774 (but that line could be pushed into the filedownloadinfo too tbh)